src/routes/devcards.ts

```diff
@@ -99,7 +99,12 @@ export default async function (fastify: FastifyInstance): Promise<void> {
         if (format === 'svg') {
           const pngUrl = `${process.env.URL_PREFIX}${req.originalUrl.replace('.svg', '.png')}`;
           const encodedUrl = encode(pngUrl, { useNamedReferences: true });
-          const svgString = `<svg xmlns="http://www.w3.org/2000/svg"><image href="${encodedUrl}" /></svg>`;
+          const svgString = `
+            <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+              <meta name="twitter:card" content="summary_large_image">
+              <meta name="twitter:title" content="DevCard of @${user.username}">
```

Comment:
Meta works inside svg? 👀
Happy to try, but wondering if it's supported like this.

Is there maybe some other tool we can test hardcoded HTML in?

Reply:
This is my question as well. Should we instead return html from the OG link with meta tags embedded in? Then we don't need to modify the API...

Reply:
Even the spec says, it should be an HTML page instead: https://ogp.me/

Reply:
Agree with what @kopancek and @idoshamun said, never seen this approach personally.

Reply:
We can do what those with fake ads on Twitter does 😈
They check the user-agent header, if it's a browser, just return the image, otherwise wrap it in a html block
![image](https://github.com/dailydotdev/daily-api/assets/1681525/d1c72452-e0fa-44c1-969d-d5d77721e1c3)